lib/livebook_web/live/session_live/secrets_component.ex

```diff
@@ -18,7 +18,7 @@ defmodule LivebookWeb.SessionLive.SecretsComponent do
       else
         assign(socket,
           data: %{"name" => prefill_form, "value" => "", "store" => "session"},
-          errors: [{"value", {"can't be blank", []}}],
+          errors: [value: {"can't be blank", []}],
```

Comment:
fwiw, we shouldn't need this workaround in the first place. I feel like we should redesign the form internals in a bigger way, i.e. instead of having `@data` and `@errors`, we should have `@changeset`. And then we could:

```diff
- <%= text_input(f, :value, value: @data["value"]) %>
+ <%= text_input(f, :value) %>
```

Reply:
I agree with that, because this form is only used for `Secret` embedded schema